test/.eslintrc

```diff
@@ -0,0 +1,3 @@
+env:
```

Comment:
no that's not true, you can add your "globlas" in the main file so that the tests file will pass the eslinting 😅  am I missing something here?

example:

https://github.com/RafalWilinski/express-status-monitor/blob/master/.eslintrc.json#L3-L9